config.json

```diff
@@ -381,6 +381,17 @@
         "filtering",
         "strings"
       ]
+    },
+    {
+      "slug": "fizz-buzz",
+      "uuid": "b35ca909-3389-4427-810f-775aa174001f",
+      "core": false,
+      "unlocked_by": null,
```

Comment:
How about letting this be unlocked by `"raindrops"`? Or the other way round? It seems [quite similar](https://github.com/exercism/r/tree/master/exercises/raindrops) content-wise.